crates/era-cheatcodes/tests/src/cheatcodes/RollFork.t.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Test, console2 as console} from "../../lib/forge-std/src/Test.sol";
+import {Constants} from "./Constants.sol";
+
+contract RollForkTest is Test {
+    uint256 mainnetFork;
+
+    function setUp() public {
+        (bool success, bytes memory data) = Constants.CHEATCODE_ADDRESS.call(
+            abi.encodeWithSignature("createFork(string)", "mainnet")
+        );
+        require(success, "createFork failed");
+
```

Comment:
just nit: seems we have to merge `refactoring` before merging new tests or use `vm.function` semantic already

Reply:
Yeah that PR should go in shortly, so this can be adapted. Alternatively this can be rebased onto the refactor PR